frontend/src/pages/videos/VideoAnalysisPage.tsx

```diff
@@ -23,12 +33,15 @@ export const VideoAnalysis = ({
 }) => {
   const { t } = useTranslation();
   const { baseUrl } = useCurrentPoll();
+  const [expanded, setExpanded] = React.useState(false);
 
   const uid = `yt:${video.video_id}`;
   const actions = useLoginState() ? [CompareNowAction, AddToRateLaterList] : [];
 
   const { criteria_scores: criteriaScores } = video;
   const shouldDisplayCharts = criteriaScores && criteriaScores.length > 0;
+  const options = { defaultProtocol: 'https' };
+  const linki = linkifyHtml(video.description || '', options);
```

Comment:
Same here, `linki` does not reflect what is the linkified text. It could be better to name it from what it represent than just one it its property